python/whylogs/core/utils/utils.py

```diff
@@ -32,6 +32,19 @@ def wrapper(*args, **kwargs):
     return deprecated_decorator
 
 
+def deprecated_argument(arg_name: str, message: Optional[str] = None):
+    def deprecated_decorator(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            if arg_name in kwargs:
+                warnings.warn(f"Argument {arg_name} is deprecated, {message if message else 'do not use it'}.")
```

Comment:
```suggestion
                warnings.warn(f"Argument {arg_name} is deprecated, {message if message else 'it will be removed in the next major version of whylogs'}.")
```